README.adoc

```diff
@@ -16,15 +16,29 @@ This repository contains a collection of tools and scripts to support the mainte
 
 === Check out Maven source code
 
+// further documentation: https://github.com/apache/maven-sources
+
 Follow the instructions in https://maven.apache.org/scm.html[Maven SCM documentation] to check out Maven to this directory.
 
+=== Further needed tools
```

Comment:
Maybe the official Maven doc is a better place

Reply:
> Maybe the official Maven doc is a better place

For me its clear so far. I guess we have to re-work this tools/docs frequently. Perhaps we find a better description in the future?

Reply:
I was thinking about persons with less experience in the Java ecosystem